prov/gni/test/cntr.c

```diff
@@ -98,6 +98,7 @@ static inline void cntr_setup_eps(void)
 	hints->mode = ~0;
 
 	hints->fabric_attr->name = strdup("gni");
+	hints->caps = GNIX_EP_RDM_PRIMARY_CAPS;
```

Comment:
Enabling FI_RMA_EVENT causes this test to hang.  I believe that this is because thread 0 (receiver of RMA from all other threads) does not progress it's NIC in any way.  RMA event control messages that are never processed fill up the mailboxes at thread 0.